spec/fabricators/source_fabricator.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 Fabricator(:source) do
-  name 'MyWebsite'
+  name 'Google'
```

Comment:
```suggestion
  name { ['Google', 'Bing'].sample }
```

This allow test to be exposed to more cases (unless the test explicitly declares which source to use) and therefore identifies some issues that could stay hidden otherwise. 

Also, when reading the Fabricator, we get a better insight about what are the supported values :)